templates/addons/s3/s3.yml

```diff
@@ -0,0 +1,13 @@
+{{.BucketResourceName}}:
```

Comment:
We might have to sanitize this.

Reply:
https://github.com/aws/amazon-ecs-cli-v2/blob/698d79d7cc19d227d1789c0d3eaa28fedcec4eab/internal/pkg/deploy/cloudformation/stack/template_functions.go#L24

Which might mean we need to move these functions to templates/functions.go